webknossos/webknossos/dataset/layer.py

```diff
@@ -360,12 +360,26 @@ def bounding_box(self, bbox: NDBoundingBox) -> None:
         """
         Updates the offset and size of the bounding box of this layer in the properties.
         """
+        self._set_bounding_box(bbox)
```

Comment:
The bounding_box setter should probably not call the `_set_bounding_box` method with it's default `expand_only = True`. Either we change the default as @philippotto suggested or we call it with `expand_only = False`:

```suggestion
        self._set_bounding_box(bbox, expand_only=False)
```

Reply:
That is a good point. Maybe we should just keep both in sync. It only affects metadata. Chunks outside the bbox are not deleted.